ec2net-functions

```diff
@@ -40,14 +40,21 @@ config_file="/etc/sysconfig/network-scripts/ifcfg-${INTERFACE}"
 route_file="/etc/sysconfig/network-scripts/route-${INTERFACE}"
 route6_file="/etc/sysconfig/network-scripts/route6-${INTERFACE}"
 dhclient_file="/etc/dhcp/dhclient-${INTERFACE}.conf"
+MAINROUTETABLE="yes"
+
 
-# make no changes to unmanaged interfaces
 if [ -s ${config_file} ]; then
+  # make no changes to unmanaged interfaces
   unmanaged=$(LANG=C grep -l "^[[:space:]]*EC2SYNC=no\([[:space:]#]\|$\)" $config_file)
   if [ "${config_file}" == "${unmanaged}" ]; then
     logger --tag ec2net "Not managing ${INTERFACE}"
     exit
   fi
+  # check to see if we should add a default route through ${INTERFACE} to the main kernel route table
+  skip_add_default_route_to_main_table=$(LANG=C grep -l "^[[:space:]]*MAINROUTETABLE=no\([[:space:]#]\|$\)" $config_file)
```

Comment:
The naming of that variable seems overly complex. Can we stick to something shorter. no_default_route? no_main_route?

Reply:
I modified the name to `no_default_route` as suggested